requirements/dev.in

```diff
@@ -1,14 +1,14 @@
 # Dependencies needed only for development/testing.
 pytest-cov
-django-debug-toolbar==2.2
+django-debug-toolbar
 mock
 responses
 django-extensions
 pytest-selenium
 PyPOM
 
 # for git commit hooks
-pre-commit
+pre-commit==2.2.0
```

Comment:
Can you please elaborate?

For the record, we don't need `pre-commit` within this file.

Reply:
we need this, it is part of the dev environment.

Reply:
`pre-commit` is recommended to be installed globally, otherwise, you need to active the environment every time you want `git commit` to trigger the pre commit hooks.

`pre-commit`, `tox` and `pip-tools` are requirements for the project rather than dependencies of it. You can keep them if you insist but this is not the logical place they should be placed in.